tutorial/markdown/kotlin/ktor.md

```diff
@@ -102,195 +95,356 @@ fun createScope(bucket: Bucket, configuration: CouchbaseConfiguration): Scope {
 
 Configured database objects like the bucket and scope must exist on the cluster prior to starting the application.
 
+### Setup Database Configuration
+
+To know more about connecting to your Capella cluster, please follow the [instructions](https://docs.couchbase.com/cloud/get-started/connect.html).
+
+Specifically, you need to do the following:
+
+- Create the [database credentials](https://docs.couchbase.com/cloud/clusters/manage-database-users.html) to access the travel-sample bucket (Read and Write) used in the application.
+- [Allow access](https://docs.couchbase.com/cloud/clusters/allow-ip-address.html) to the Cluster from the IP on which the application is running.
+
+All configuration for communication with the database is stored in the `src/main/resources/application.conf` file under the `couchbase` section:
+
+```
+couchbase {
+    connectionString = "localhost"
```

Comment:
Does it work with just localhost without couchbase://?

Reply:
Corrected it.